accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               </ol>
             </div>
 
+            <h3 id="a11y_considerations">6.5 Accessibility Considerations</h3>
+            <div className="pa">
+               Highlighted below are several accessibility features for adaptability and to ensure ease of access to the Checker functionality with keyboard or with a screen reader: 
+              <ol
+                style={{ listStyleType: "decimal", marginInlineStart: "2rem" }}
+              >
+                <li>The Accessibility Checker tool is responsive to user's preferred font size and colors.
+                </li>
+                <li>Both the Accessibility Assessment view and the Accessibility Checker view are fully keyboard accessible, navigate as follows:
+                </li>
+                  <ul
+                    style={{
+                      listStyleType: "circle",
+                      marginInlineStart: "2rem",
+                    }}
+                  >
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Use the <strong>'tab'</strong> key to navigate to any focusable element in the checker, starting
+                        with the <strong>'Scan'</strong> button once the checker launches.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        After running the scan, press the <strong>'tab'</strong> key again, to navigate to the <strong>Reset Selections</strong> icon and the <strong>Report</strong> icon.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the checkbox by each issue type and press the <strong>'Enter'</strong> key to filter the list of issue by <strong>Violations</strong>, <strong>Needs review</strong> and/or by <strong>Recommendations</strong>.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the <strong>Issue List</strong> tabs and use the <strong>'right'</strong> or the <strong>'left'</strong> arrow keys to navigate between the <strong>Element Roles</strong> view, the <strong>Checkpoints</strong> view and the <strong>Rules</strong> view.
```

Comment:
Again, should Checklist, Element roles and Rules have quotes around them to be consistent with 'Scan'?